share/functions/__fish_print_ninja_tools.fish

```diff
@@ -0,0 +1,4 @@
+function __fish_print_ninja_tools
+    echo list
+    ninja -t list | grep -v ':' | sed -Ee 's/[[:space:]]+([^[:space:]]+).*/\1/'
```

Comment:
As above, `string` is your friend here - `string match -v ':' | string replace -r '\s+(\S+).*' '$1'` (untested).

The "-E" option to sed is problematic as it's completely undocumented for GNU sed (though it appears to work).